src/fu-hwids.c

```diff
@@ -276,6 +276,28 @@ fu_hwids_convert_base10_integer_cb (FuSmbios *smbios,
 	return g_strdup_printf ("%u", data_raw[offset]);
 }
 
+static gchar *
+fu_hwids_convert_base10_integer_padded_cb (FuSmbios *smbios,
+				    guint8 type, guint8 offset,
```

Comment:
Please make sure that these line up to the line above (see the function copied from for an example)

Reply:
done